starter-code/pizza.js

```diff
@@ -1 +1,39 @@
 // Write your Pizza Builder JavaScript in this file.
+function main() {
+  resetPizza('.btn-crust', '.crust-gluten-free');
+  // toggleIngredients('.btn-sauce', '.sauce-white');
+}
+
+$('.btn-crust').toggleClass('active');
+$('section.crust').toggleClass('crust-gluten-free');
+
+$('.btn-pepperonni').on('click', toggleIngredients('.btn-pepperonni', '.pep'));
```

Comment:
You DON'T need to attach the function to the button two times: one in this line, another in the function toggleIngredients.